docs/pages/versions/unversioned/sdk/local-authentication.md

```diff
@@ -64,7 +64,9 @@ Attempts to authenticate via Fingerprint/TouchID (or FaceID if available on the
 
 - **options (_object_)** -- An object of options.
   - **promptMessage (_string_)** -- A message that is shown alongside the TouchID or FaceID prompt. (**iOS only**)
-  - **fallbackLabel (_string_)** -- Allows to customize the default `Use Passcode` label shown after several failed authentication attempts. Setting this option to an empty string disables fallback to device passcode. (**iOS only**)
+  - **cancelLabel (_string_)** -- Allows to customize the default `Cancel` label shown. (**iOS only**)
+  - **fallbackLabel (_string_)** -- Allows to customize the default `Use Passcode` label shown after several failed authentication attempts. Setting this option to an empty string disables this button from showing in the prompt. (**iOS only**)
+  - **disableSystemPasscodeAsFallback (_bool_)** -- After several failed attempts the system will fallback to the device passcode.  This setting allows you to disable this option and instead handle the fallback yourself.  This can be preferable in certain custom authentication work flows.  This behaviour maps to using the iOS [LAPolicyDeviceOwnerAuthenticationWithBiometrics](https://developer.apple.com/documentation/localauthentication/lapolicy/lapolicydeviceownerauthenticationwithbiometrics?language=objc) policy rather than the [LAPolicyDeviceOwnerAuthentication](https://developer.apple.com/documentation/localauthentication/lapolicy/lapolicydeviceownerauthentication?language=objc) policy (**iOS only**)
```

Comment:
Also, I think we're able to support this option on Android too, as of Android 10. We just need to change the parameter of `setDeviceCredentialAllowed`:
https://github.com/expo/expo/blob/f9f841f8f63a6086551ce577995c2cef32717fc2/packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.java#L126-L128

Then I'd suggest renaming this option since on Android the default authentication can be different than passcode, it could be a pattern or password as well (depending on user's settings). Maybe `disableDeviceFallback`? Could you then add this option to TypeScript typings?